### PR TITLE
feat(ci): add suite-native android build and deploy develop workflow

### LINKED ISSUE #5429: suite-native: Setup mobile app build for Android with Github actions
- Setup CI build using created fastlane lanes for Android
- use Github CI

Related to https://github.com/trezor/trezor-suite/issues/5521

### PR DESCRIPTION
**Pull request by vdovhanych**
Closes https://github.com/trezor/trezor-suite/issues/5429

### CODE REVIEW

File: .github/workflows/suite-native_ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+      - name: Decode files
+        env:
+          ENCODED_STRING_KEYSTORE: ${{ secrets.DEVELOP_KEYSTORE_FILE }}
+          ENCODED_STRING_JSON_FILE: ${{secrets.GOOGLE_PLAY_JSON_KEY}}
+        run: |
+          TMP_FILE_PATH="${RUNNER_TEMP}/app"
+          mkdir "${TMP_FILE_PATH}"
+          echo $ENCODED_STRING_KEYSTORE | base64 -di > "${TMP_FILE_PATH}/release.keystore"
+          edho $ENCODED_STRING_JSON_FILE > "${TMP_FILE_PATH}/firebase_key.json"
```
Comment (bouchja1) [inline .github/workflows/suite-native_ci.yml]:
```suggestion
          echo $ENCODED_STRING_JSON_FILE > "${TMP_FILE_PATH}/firebase_key.json"
```
Sorry about nitpicking in draft. I'll just keep you from being sad after the first try :). Same for [suite_native_production_ci.yml](https://github.com/trezor/trezor-suite/pull/5676/files#diff-e5dbd3cb385c674a2ba5fa9b7b83d333030bea82e7383d49d1e7f221fd0b52c3)

Comment (vdovhanych) [reply .github/workflows/suite-native_ci.yml]: Thnx 😆

**Comment by Nodonisko** [comment]
What is missing? Why it is draft?

**Comment by vdovhanych** [comment]
please check the directories and the build scripts if that is ok, let's merge this and test how it works. I needed to set up some environments but otherwise, it should be done.

**Comment by bouchja1** [comment]
> please check the directories and the build scripts if that is ok, let's merge this and test how it works. I needed to set up some environments but otherwise, it should be done.

Seems good to me.

**Review by bouchja1 — APPROVED** [review]

**Review by juriczech — APPROVED** [review]

**Pull request merged**